lib/cocoapods/installer/xcode/pods_project_generator/file_references_installer.rb

```diff
@@ -71,6 +71,7 @@ def refresh_file_accessors
           # Prepares the main groups to which all files will be added for the respective target
           #
           def prepare_pod_groups
+            # either handle a new type of file_accessor for e.g. "subdirectory_readme" or, if they're included in a preexisting file_accessor, add logic to account for that here?
```

Comment:
dont think anything needs to change here - you can look at the `install!` method in this file to see other examples.

the existing logic for the local pod files is there, and is called `add_developer_files` - if you're proposing we do that for all pods (not just local ones) you could update `add_developer_files` to include your desired functionality and also remove the check for whether the pod is local